### ISSUE TITLE
MODEL-ZOO: Spinenet-190 Mask-RCNN ckpt is missing

### ISSUE THREAD

**Issue by mitraavi** [tensorflow/tpu#781]
Tried to download Spinenet-190 Mask-RCNN ckpt from MODEL-ZOO, I'm getting following error:

No such object: cloud-tpu-checkpoints/detection/maskrcnn/spinenet190-mrcnn-pc.tar.gz

**Comment by xianzhidu** [comment]
Please use this link: 
https://storage.cloud.google.com/cloud-tpu-checkpoints/detection/maskrcnn/spinenet-190.tar.gz

**Comment by mitraavi** [comment]
Dear Mr. Xianzhi,

I am writing to thank you!... for your generous reply with a link to your
pretrained model.

It allows me to explore your research work and gain knowledge around it.

Thanking You!

Best regards,
Abhishek
